src/System.Private.ServiceModel/src/System/ServiceModel/Channels/HttpChannelFactory.cs

```diff
@@ -513,18 +500,12 @@ protected override IAsyncRequest CreateAsyncRequest(Message message)
                 return new HttpClientChannelAsyncRequest(this);
             }
 
-            public virtual Task<HttpRequestMessage> GetHttpRequestMessageAsync(EndpointAddress to, Uri via, CancellationToken cancelToken)
+            internal Task<HttpRequestMessage> GetHttpRequestMessageAsync(EndpointAddress to, Uri via, TimeoutHelper timeoutHelper)
             {
                 // This method replaces the method with the following prototyp:
                 //      protected HttpWebRequest GetWebRequest(EndpointAddress to, Uri via, SecurityTokenContainer clientCertificateToken, ref TimeoutHelper timeoutHelper)
                 // SecurityTokenContainer doesn't exist so was excluded from method signature
-                return Task.FromResult(this.Factory.GetHttpRequestMessage(to, via, cancelToken, false));
-            }
-
-
-            public virtual bool WillGetWebRequestCompleteSynchronously()
-            {
-                return true;
+                return Task.FromResult(this.Factory.GetHttpRequestMessage(to, via));
             }
```

Comment:
Nit: sometime the code uses 'this.Factory' and sometimes just 'Factory'.  Recommend be consistent throughout with 'this'.